src/shared/error-boundary/error-boundary.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class ErrorBoundary extends React.Component<
+  {},
+  { error: string; errorInfo: string }
+> {
+  constructor(props) {
+    super(props);
+    this.state = { error: null, errorInfo: null };
+  }
+
```

Comment:
You should also implement `getDerivedStateFromError` method
[https://ru.reactjs.org/docs/error-boundaries.html](url)